CI/physmon/phys_perf_mon.sh

```diff
@@ -66,5 +66,25 @@ run \
     -p $outdir/truth_tracking_plots \
 
 
+echo "::group::full_chain_odd"
+
+thirdparty/OpenDataDetector/ci/full_chain_odd.py -o $outdir
+
+build/bin/ActsAnalysisResidualsAndPulls \
+    --predicted --filtered --smoothed --silent \
+    -i $outdir/trackstates_ckf.root \
+    -o $outdir/acts_analysis_residuals_and_pulls.root \
+
+histcmp \
```

Comment:
If you use the same `run` bash function that's used for the other comparisons, you won't have to duplicate the error code handling.

Reply:
good point thanks!